dbt_copilot_helper/commands/conduit.py

```diff
@@ -152,6 +153,33 @@ def connect_to_addon_client_task(app: str, env: str, cluster_arn: str, addon_nam
         raise CreateTaskTimeoutConduitError
 
 
+def add_stack_delete_policy_to_task_role(app: str, env: str, addon_type: str):
+    conduit_stack_name = f"task-conduit-{app}-{env}-{app}-{addon_type}"
+    conduit_stack_resources = boto3.client("cloudformation").list_stack_resources(
+        StackName=conduit_stack_name
+    )["StackResourceSummaries"]
+
+    for resource in conduit_stack_resources:
+        if resource["LogicalResourceId"] == "DefaultTaskRole":
+            task_role_name = resource["PhysicalResourceId"]
+            boto3.client("iam").put_role_policy(
+                RoleName=task_role_name,
+                PolicyName="DeleteCloudFormationStack",
+                PolicyDocument=json.dumps(
+                    {
+                        "Version": "2012-10-17",
+                        "Statement": [
+                            {
+                                "Action": ["cloudformation:DeleteStack"],
+                                "Effect": "Allow",
+                                "Resource": f"arn:aws:cloudformation:*:*:stack/{conduit_stack_name}/*",
+                            }
+                        ],
+                    }
+                ),
+            )
```

Comment:
Just for consistency

```suggestion
            boto3.client("iam").put_role_policy(
                RoleName=task_role_name,
                PolicyName="DeleteCloudFormationStack",
                PolicyDocument=json.dumps(
                    {
                        "Version": "2012-10-17",
                        "Statement": [
                            {
                                "Action": ["cloudformation:DeleteStack"],
                                "Effect": "Allow",
                                "Resource": f"arn:aws:cloudformation:*:*:stack/{conduit_stack_name}/*",
                            },
                        ],
                    },
                ),
            )
```